gr-blocks/python/blocks/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ GR_PYTHON_INSTALL(
     msg_pair_to_var.py
     msg_meta_to_pair.py
     var_to_msg.py
+    pdu_compatibility.py # REMOVE IN 3.11
```

Comment:
You're effectively deprecating something, and announcing it for later removal. Since this can be hard to track, please also append to CHANGELOG.md.

Reply:
Agree, fixed.